company_today/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: 2022 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+{
+    "name": "Company Today",
+    "summary": """
+        Store today's date on the company model.""",
+    "version": "16.0.1.0.0",
+    "category": "Uncategorized",
+    "website": "https://github.com/coopiteasy/addons",
+    "author": "Coop IT Easy SC",
+    "license": "AGPL-3",
+    "application": False,
+    "depends": [
+        "base",
+    ],
+    "excludes": [],
+    "data": [
+        "data/ir_cron.xml",
+    ],
+    "demo": [],
+    "qweb": [],
```

Comment:
```suggestion
    "depends": [
        "base",
    ],
    "data": [
        "data/ir_cron.xml",
    ],
```

default and empty values should ideally be removed.